lua/catppuccin/types.lua

```diff
@@ -222,6 +222,8 @@
 ---@class CtpIntegrationIndentBlankline
 -- Whether to enable the integration.
 ---@field enabled boolean
+-- Sets the color of the scope line
+---@field scope_color string?
```

Comment:
Oh sorry I missed that this [*has to be a Catppuccin color*](https://github.com/catppuccin/nvim/blob/2fe1aa6a19ae6b5346da3ee0211d87685b6faf7d/lua/catppuccin/groups/integrations/indent_blankline.lua#L10) before.
Let's please use
```suggestion
---@field scope_color CtpColor?
```
Should be good to merge after that.